crates/turborepo-errors/src/lib.rs

```diff
@@ -113,6 +113,12 @@ impl<T: Deserializable> Deserializable for Spanned<T> {
     }
 }
 
+impl Spanned<String> {
+    pub fn as_str(&self) -> &str {
+        self.value.as_str()
+    }
+}
+
```

Comment:
I don't think this is needed since `Spanned` already implements deref?
```suggestion
```

Reply:
I think you need to specify the deref type sometimes to help out. I know I've needed to do that when working with `Cow` sometimes